cost-analyzer/templates/federated-store-secret.yaml

```diff
@@ -1,12 +0,0 @@
-{{- if .Values.kubecostModel.federatedStorageConfig -}}
```

Comment:
Why are we deleting the `federated-store` Secret? The Aggregator and CloudCosts containers still need this right?

Reply:
Oh, or did you intend to replace this with the `.Values.kubecost.exportBucket`?

Reply:
yes something like that. this secret is shared use with the finops-agent subchart so I am trying to move us away from current configuration paradigm to something that works a little bit better